docs/application/tizen-studio/setup/prerequisites.md

```diff
@@ -4,7 +4,7 @@ Check the following prerequisites before attempting to install the Tizen Studio.
 
 ## Java Development Kit (JDK) Requirements
 
-You must install Oracle Java Development Kit (JDK) 8 for Tizen Studio 3.3 or below or OpenJDK 12 for Tizen Studio 3.5 or above.
+You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 for Tizen Studio 3.5 or above.
```

Comment:
Suggestion:

You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 for Tizen Studio 3.5 and higher.